src/libraries/apiclient/apiclientcore.js

```diff
@@ -244,24 +263,24 @@ define(["events", "appStorage"], function(events, appStorage) {
         var lowered = url.toLowerCase();
         return "/" !== name.charAt(0) && (url += "/"), url += name, params && (params = paramsToString(params)) && (url += "?" + params), url
     }, ApiClient.prototype.fetchWithFailover = function(request, enableReconnection) {
-        console.log("Requesting " + request.url), request.timeout = 3e4;
+        console.debug("Requesting " + request.url), request.timeout = 3e4;
         var instance = this;
         return getFetchPromise(request).then(function(response) {
             return instance.lastFetch = (new Date).getTime(), response.status < 400 ? "json" === request.dataType || "application/json" === request.headers.accept ? response.json() : "text" === request.dataType || 0 === (response.headers.get("Content-Type") || "").toLowerCase().indexOf("text/") ? response.text() : response : (onFetchFail(instance, request.url, response), Promise.reject(response))
         }, function(error) {
-            if (error ? console.log("Request failed to " + request.url + " " + (error.status || "") + " " + error.toString()) : console.log("Request timed out to " + request.url), error && error.status || !enableReconnection) throw console.log("Reporting request failure"), onFetchFail(instance, request.url, {}), error;
-            console.log("Attempting reconnection");
+            if (error ? console.error("Request failed to " + request.url + " " + (error.status || "") + " " + error.toString()) : console.error("Request timed out to " + request.url), error && error.status || !enableReconnection) throw console.error("Reporting request failure"), onFetchFail(instance, request.url, {}), error;
```

Comment:
I wonder how many more of those uglified things are still in the codebase...

Reply:
Apiclient seems to be the main offender. So hopefully, once we're able to take it out of the tree and use the JS Apiclient repo, there should be a lot less of these.

Still a bunch of ternary operators that need cleanup, as well as usage of the comma operator...